src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -290,7 +315,11 @@ public IEnumerable<ImageComboBoxOption> GetAvailableRowsFor(ColumnMappingDTO mod
          {
             var metaDataCategory = _metaDataCategories.FirstOrDefault(md => md.Name == model.MappingName);
             if (metaDataCategory != null && metaDataCategory.ShouldListOfValuesBeIncluded)
+            {
+               if (!metaDataCategory.ListOfValues.Values.Contains(model.ExcelColumn))
+                  options.Add(new ImageComboBoxOption() { Description = model.ExcelColumn, ImageIndex = ApplicationIcons.IconIndex(ApplicationIcons.MetaData) });
```

Comment:
What is ImageComboBoxOption? Is it a devexpress control?

Reply:
It is an entity defined in OSPSuite.Presentation.Views.Importer holding information for creating the image combo box (label, icon)

Reply:
ok. Unsual to see the name of a user control here. What if it becomes a list or sthg. Typically this is what we call a DTO